Sources/SystemInternals/WindowsSyscallAdapters.swift

```diff
@@ -35,6 +38,7 @@ internal func close(_ fd: Int32) -> Int32 {
 internal func lseek(
   _ fd: Int32, _ off: off_t, _ whence: Int32
 ) -> off_t {
+  // FIXME(compnerd) should this use `_lseeki64` to enable large file access?
```

Comment:
On Darwin, the `off_t` is a 64bit int. Is Windows different?

Reply:
Yeah, windows and Linux both do separation of 32-bit and 64-bit.  The off_t is 32-bit.

Reply:
I'm in favor of always using the 64-bit variation